### ISSUE TITLE
Clicking the avatar on web settings/player should show options

### ISSUE THREAD

**Issue by SubJunk** [UniversalMediaServer/UniversalMediaServer#4857]
A first step would be that it shows a logout option. An eventual step would be a user switcher, because that is what I think users would expect based on other services.

**Comment by Mik-S-UMS** [comment]
It would make sense to be able to access account settings here too.

**Comment by SubJunk** [comment]
Good idea. Yes it appears clickable and is confusing when nothing happens on click

**Comment by SubJunk** [comment]
This was done in v15
